src/pages/CreateFeedback.tsx

```diff
@@ -0,0 +1,202 @@
+          required
+          className="text"
+          value={title}
+          onChange={changeTitle}
+        />
+
+        <label htmlFor="category" className="create-feedback_heading">
+          <span>Category</span>
+          <span>Choose a category for your feedback</span>
+        </label>
+
+        <ul className="dropdown">
+          <div onClick={openDropDown} className={openBorder}>
+            <li
+              role="button"
+              aria-labelledby="dropdown-label"
+              id="dropdown__selected"
+              tabIndex={0}
+            >
+              {category}
+            </li>
+            <img src={isOpen ? arrowUp : arrowDown} alt="arrow down" />
+          </div>
+
+          <li
+            aria-expanded={isOpen}
+            role="list"
+            className="dropdown__list-container"
+          >
+            {isOpen === false ? null : (
+              <ul className="dropwdown_list">
+                <li className="dropdown_list-item" tabIndex={0}>
+                  <span
+                    id="Feature"
+                    className={category === 'Feature' ? 'currentCategory' : ''}
+                    onClick={changeCategory}
+                  >
+                    Feature
+                  </span>
+                </li>
+                <li className="dropdown_list-item" tabIndex={0}>
+                  <span
+                    id="Enhancement"
+                    className={
+                      category === 'Enhancement' ? 'currentCategory' : ''
+                    }
+                    onClick={changeCategory}
+                  >
+                    Enhancement
+                  </span>
+                </li>
+                <li className="dropdown_list-item" tabIndex={0}>
+                  <span
+                    id="Bug"
+                    className={category === 'Bug' ? 'currentCategory' : ''}
+                    onClick={changeCategory}
+                  >
+                    Bug
+                  </span>
+                </li>
+                <li
+                  className="dropdown_list-item"
+                  tabIndex={0}
+                  id="UI"
+                  onClick={changeCategory}
+                >
+                  <span
+                    id="UI"
+                    className={category === 'UI' ? 'currentCategory' : ''}
+                    onClick={changeCategory}
+                  >
+                    UI
+                  </span>
+                </li>
+                <li className="dropdown_list-item" tabIndex={0}>
+                  <span
+                    id="UX"
+                    className={category === 'UX' ? 'currentCategory' : ''}
+                    onClick={changeCategory}
+                  >
+                    UX
+                  </span>
+                </li>
+              </ul>
+            )}
+          </li>
+        </ul>
+
+        <label htmlFor="title" className="create-feedback_heading">
```

Comment:
Where is the `htmlFor` being used? It should be tied to something.

Reply:
This needs to be tied to the `textarea` still?

Reply:
Updated